File: array.js
// ARRAY CRUD
const students = ['Michelle', 'Raj', 'Hamish'];

// Create
students.push("Ethan");
console.log(students);

// Read
console.log(students[1]);

// Update
students[0] = "Super Michelle";
console.log(students);

// Delete
students.splice(0,1);
console.log(students);

// Iterate
// () => {}  JS arrow function
students.forEach((student) => {
  console.log(`${student} is amazing!`);
});

File: array.rb
# ARRAY CRUD
students = ['Michelle', 'Raj', 'Hamish']

# Create
students << "Ethan"
students.push("Tyler")
p students

# Read
# p students.second # only Rails
p students[1]

# Update
students[0] = "Super Michelle"
p students

# Delete
students.delete_at(0)
students.delete('Raj')
p students

# Iterate
students.each do |student|
  puts "#{student} is amazing!"
end